test/TestUtils.ts

```diff
@@ -34,19 +41,14 @@ export function stubCatalogFixture(): void {
 	});
 }
 
-/**
- * Initial setup needed by all oclif command unit tests.
- *
- * Example:
- * setupCommandTest
- * 	.command(['scanner:run', '-t', 'test-code'])
- * 	.it('Scanner Run Relative Path Succeeds', ctx => {
- * 		expect(ctx.stdout).to.contain('No rule violations found.');
- * 	});
- */
-export const setupCommandTest = test
-	.do(() => TestOverrides.initializeTestSetup())
-	.stdout()
-	.stderr();
+export function runCommand(command: string): ExecCmdResult<AnyJson> {
+	TestOverrides.initializeTestSetup();
+	return execCmd(command);
+}
+
+export function runInteractiveCommand(command: string, answers: PromptAnswers): Promise<InteractiveCommandExecutionResult> {
```

Comment:
This method is technically unused for now. Interactivity seems to require `SfCommand`, so we have to wait until then to fully use this.

Reply:
Out of curiosity, do we have interactive commands that prompt for user input? If so, do we not already have tests for them today?

Reply:
`scanner rule remove` lists the rules that are going to be removed and prompts the user to confirm. We had tests in `remove.test.ts` that checked this functionality, but those had to be disabled for now.